src/Monitor.vala

```diff
@@ -100,5 +108,20 @@ namespace Monitor {
             return app.run (args);
         }
 
+        public static void load_css_provider (){
+            var custom_css = "";
+            var gtk_settings = Gtk.Settings.get_default ();
+
+            if (gtk_settings.gtk_application_prefer_dark_theme == true) {
```

Comment:
If this key is a Boolean, you can check for it directly. No need to `== true`. Also, it might be shorter to use a ternary conditon here.